src/components/common/Video.js

```diff
@@ -3,20 +3,23 @@ import {
   TouchableOpacity,
   TouchableWithoutFeedback,
   Image,
-  View
+  View,
+  Dimensions
 } from 'react-native';
 import { Video } from 'expo';
 import playIcon from '../../assets/images/play.png';
-const replayIcon = playIcon;
 
+const replayIcon = playIcon;
+var res = Dimensions.get('window');
 const styles = {
   container: {
     width: '100%',
     alignItems: 'center',
     justifyContent: 'center'
   },
   video: {
-    width: '100%'
+    width: res.width,
+    height: 0.3*res.height
```

Comment:
Is `height` needed? If so, `Video` should accept the aspect ratio of the video as prop.

Reply:
From what I've tried so far (at least on my mobile device), both parameters are necessary for the video to be visible from the beginning (rendering of the component).

Reply:
Then I would suggest passing the aspect ratio